packages/nodes-base/nodes/MessageBird/MessageBird.node.ts

```diff
@@ -0,0 +1,354 @@
+					show: {
+						resource: ['sms']
+					}
+				},
+				options: [
+					{
+						name: 'Send',
+						value: 'send',
+						description: 'Send text messages (SMS)'
+					}
+				],
+				default: 'send',
+				description: 'The operation to perform.'
+			},
+
+			// ----------------------------------
+			//         sms:send
+			// ----------------------------------
+			{
+				displayName: 'From',
+				name: 'originator',
+				type: 'string',
+				default: '',
+				placeholder: '14155238886',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: ['send'],
+						resource: ['sms']
+					}
+				},
+				description: 'The number from which to send the message'
+			},
+			{
+				displayName: 'To',
+				name: 'recipients',
+				type: 'string',
+				default: '',
+				placeholder: '14155238886/+14155238886',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: ['send'],
+						resource: ['sms']
+					}
+				},
+				description: 'all recipients separated by commas'
+			},
+
+			{
+				displayName: 'Message',
+				name: 'message',
+				type: 'string',
+				default: '',
+				required: true,
+				displayOptions: {
+					show: {
+						operation: ['send'],
+						resource: ['sms']
+					}
+				},
+				description: 'The message to be send'
+			},
+			{
+				displayName: 'Additional Fields',
+				name: 'additionalFields',
+				type: 'collection',
+				placeholder: 'Add Fields',
+				default: {},
+				options: [
+					//date-time format
+					{
+						displayName: 'Created Date-time',
+						name: 'createdDatetime',
+						type: 'dateTime',
+						placeholder: '2011-08-30T09:30:16.768-04:00',
+						default: '',
+						description:
+							'The date and time of the creation of the message in RFC3339 format (Y-m-dTH:i:sP).'
+					},
+					{
+						displayName: 'Datacoding',
+						name: 'datacoding',
+						type: 'string',
+						default: '',
+						description:
+							'Using unicode will limit the maximum number of characters to 70 instead of 160'
+					},
+					{
+						displayName: 'Gateway',
+						name: 'gateway',
+						type: 'number',
+						default: '',
+						description: 'The SMS route that is used to send the message.'
+					},
+					{
+						displayName: 'Group Ids',
+						name: 'groupIds',
+						placeholder: '1,2',
+						type: 'string',
+						default: '',
+						description:
+							'group ids separated by commas, If provided recipients can be omitted'
+					},
+					{
+						displayName: 'Mclass',
+						name: 'mclass',
+						type: 'options',
+						placeholder: 'permissible values from 0-3',
+						options: [
+							{
+								name: '0',
+								value: '0'
+							},
+							{
+								name: '1',
+								value: '1'
+							},
+							{
+								name: '2',
+								value: '2'
+							},
+							{
+								name: '3',
+								value: '3'
+							}
+						],
+						default: '',
+						description:
+							'Indicated the message type. 1 is a normal message, 0 is a flash message.'
```

Comment:
What are the other two types? Maybe remove the 2 and 3, we dont' want users selecting things they don't know about. Messagebird doesn't even mention them on the docs.
https://developers.messagebird.com/api/sms-messaging/